core/app/c/(public)/[communitySlug]/public/forms/[formSlug]/fill/ExternalFormWrapper.tsx

```diff
@@ -214,7 +248,7 @@ export const ExternalFormWrapper = ({
 			<form
 				onChange={formInstance.handleSubmit(handleAutoSave)}
 				onSubmit={formInstance.handleSubmit(handleSubmit)}
-				className={cn("relative flex flex-col gap-6", className)}
+				className={cn("relative isolate flex flex-col gap-6", className)}
```

Comment:
fun CSS: avoids z-index shenanigans where the little form element enablers appear above the header
![image](https://github.com/user-attachments/assets/d0a5f905-d5b5-4a0c-8bd6-b9f13d74b53a)